### PR TITLE
CB-12904 - All policy dependencies should be presented by CLI

### PR DESCRIPTION
**Pull request by gregito**

### CODE REVIEW

File: cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsCredentialConnector.java [hunk 1]
```diff
@@ -104,10 +104,22 @@ public CredentialPrerequisitesResponse getPrerequisites(CloudContext cloudContex
...
 
+    private Map<String, String> collectNecessaryPolicies() {
+        return Map.of(
+                "Log", awsPlatformParameters.getCdpLogPolicyJson(),
+                "Audit", awsPlatformParameters.getAuditPoliciesJson(),
+                "DynamoDB", awsPlatformParameters.getCdpDynamoDbPolicyJson(),
+                "Bucket access", awsPlatformParameters.getCdpBucketAccessPolicyJson(),
+                "Environment", awsPlatformParameters.getEnvironmentMinimalPoliciesJson(),
+                "Ranger audit S3", awsPlatformParameters.getCdpRangerAuditS3PolicyJson(),
+                "Datalake Admin S3", awsPlatformParameters.getCdpDatalakeAdminS3PolicyJson()
```
Comment (doktoric) [inline cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsCredentialConnector.java:119]: Datalake_Admin

File: cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsCredentialConnector.java [hunk 2]
```diff
@@ -104,10 +104,22 @@ public CredentialPrerequisitesResponse getPrerequisites(CloudContext cloudContex
...
     }
 
+    private Map<String, String> collectNecessaryPolicies() {
+        return Map.of(
+                "Log", awsPlatformParameters.getCdpLogPolicyJson(),
+                "Audit", awsPlatformParameters.getAuditPoliciesJson(),
+                "DynamoDB", awsPlatformParameters.getCdpDynamoDbPolicyJson(),
+                "Bucket access", awsPlatformParameters.getCdpBucketAccessPolicyJson(),
+                "Environment", awsPlatformParameters.getEnvironmentMinimalPoliciesJson(),
+                "Ranger audit S3", awsPlatformParameters.getCdpRangerAuditS3PolicyJson(),
```
Comment (doktoric) [inline cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsCredentialConnector.java:118]: Ranger_Audit

File: cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsCredentialConnector.java [hunk 3]
```diff
@@ -104,10 +104,22 @@ public CredentialPrerequisitesResponse getPrerequisites(CloudContext cloudContex
...
+        awsPrerequisites.setPolicies(collectNecessaryPolicies());
         return new CredentialPrerequisitesResponse(cloudContext.getPlatform().value(), accountId, awsPrerequisites);
     }
 
+    private Map<String, String> collectNecessaryPolicies() {
+        return Map.of(
+                "Log", awsPlatformParameters.getCdpLogPolicyJson(),
+                "Audit", awsPlatformParameters.getAuditPoliciesJson(),
+                "DynamoDB", awsPlatformParameters.getCdpDynamoDbPolicyJson(),
+                "Bucket access", awsPlatformParameters.getCdpBucketAccessPolicyJson(),
```
Comment (doktoric) [inline cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsCredentialConnector.java:116]: Bucket_Access

**Review by doktoric — APPROVED** [review]

File: cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/GcpPlatformParameters.java [hunk 4]
```diff
@@ -95,6 +95,11 @@ public String resourceDefinition(String resource) {
         return cloudbreakResourceReaderService.resourceDefinition("gcp", resource);
     }
 
+    @Override
+    public String resourceDefinitionInSubDir(String subDir, String resource) {
+        return cloudbreakResourceReaderService.resourceDefinitionInSubDir(subDir, "gcp", resource);
```
Comment (doktoric) [inline cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/GcpPlatformParameters.java:100]: in azure this is FileReaderUtils.readFileFromClasspathQuietly("definitions" + subDir + "/azure-" + resource + ".json");

Comment (gregito) [reply cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/GcpPlatformParameters.java:100]: you're right, but what is your point? :)

**Pull request merged**